examples/s3/fputencrypted-object.go

```diff
@@ -20,10 +20,10 @@
 package main
 
 import (
-	"log"
-
+	//"io/ioutil"
 	"github.com/minio/minio-go"
 	"github.com/minio/minio-go/pkg/encrypt"
+	"log"
```

Comment:
standard imports "log" should be placed above the public imports https://github.com/golang/go/wiki/CodeReviewComments#imports